Packs/VMRay/Integrations/VMRay/VMRay.py

```diff
@@ -1,7 +1,10 @@
-import io
-import os
 import demistomock as demisto  # noqa: F401
 from CommonServerPython import *  # noqa: F401
+
+import io
+import os
```

Comment:
Can you explain why this is necessary?

Reply:
I'm not sure why they where added.
I just copied the integration and added the argument.
I can't explain where whey come from.

Reply:
Ok I'll remove it.
Probably added by the server automatically.